astrocut/footprint_cutouts.py

```diff
@@ -51,9 +52,38 @@ def ind_sregion_to_polygon(s_reg):
 
 
 @cached(cache=FFI_TTLCACHE, lock=Lock())
-def get_caom_ffis(product: str = 'SPOC'):
+def _get_s3_ffis(s3_uri, as_table: bool = False, load_polys: bool = False):
     """
-    Fetches footprints for Full Frame Images (FFIs) from the Common Archive Observation Model. The resulting
+    Fetches footprints for Full Frame Images (FFIs) from a given S3 uri.
+
+    If as_table is specified, the return value will be an astropy table with each row an individual footprint.
+
+    If load_polys is specified, compute and return spherical_polygon Polygon objects from the points and vectors.
+
+    Parameters
+    ----------
+    as_table : bool, optional
+        Default False. Return the footprint file as an Astropy Table
+    load_polys : bool, optional
+        Default False. Convert the s_region column to an array of SphericalPolygon objects
+    """
+    # Open footprint file with fsspec
+    open_file_container = fsspec.open(s3_uri, s3={'anon': True})
+    with open_file_container as f:
+        ffis = json.load(f)
```

Comment:
Can we simplify this to:

```suggestion
    with fsspec.open(s3_uri, s3={'anon':True}) as f:
        ffis = json.load(f)
```